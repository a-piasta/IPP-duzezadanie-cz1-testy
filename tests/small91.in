newRoute;873;hccmkzuidd;xsxhfdsggagzxyoofregsr
addRoad;zzhrpgfmjyrqbdzxwxerb;kegxiwchvwecbjvjpvppuqfixwof;22;10
extendRoute;50;watdcwroxox
addRoad;kegxiwchvwecbjvjpvppuqfixwof;kegxiwchvwecbjvjpvppuqfixwof;25;-1
removeRoad;kegxiwchvwecbjvjpvppuqfixwof;zzhrpgfmjyrqbdzxwxerb
getRouteDescription;873
newRoute;643;hccmkzuidd;kegxiwchvwecbjvjpvppuqfixwof
addRoad;kdwiymevvdwewm;xsxhfdsggagzxyoofregsr;2;15
removeRoad;zzhrpgfmjyrqbdzxwxerb;kegxiwchvwecbjvjpvppuqfixwof
newRoute;690;yloqnkncpf;yloqnkncpf
getRouteDescription;690
extendRoute;873;rdebljwrwetdgqakyvnuv
newRoute;294;bjictfniafzvvefu;kegxiwchvwecbjvjpvppuqfixwof
getRouteDescription;690
addRoad;hccmkzuidd;kegxiwchvwecbjvjpvppuqfixwof;6;12
newRoute;176;rdebljwrwetdgqakyvnuv;yloqnkncpf
repairRoad;rdebljwrwetdgqakyvnuv;yloqnkncpf;-14
getRouteDescription;311
extendRoute;690;kegxiwchvwecbjvjpvppuqfixwof
repairRoad;bjictfniafzvvefu;kdwiymevvdwewm;9
removeRoad;kegxiwchvwecbjvjpvppuqfixwof;zzhrpgfmjyrqbdzxwxerb
getRouteDescription;643
extendRoute;294;kdwiymevvdwewm
newRoute;293;hccmkzuidd;zzhrpgfmjyrqbdzxwxerb
extendRoute;643;bjictfniafzvvefu
addRoad;hccmkzuidd;yloqnkncpf;16;-13
newRoute;129;yloqnkncpf;dawcsmfxldvkjmntwrrgrdknaqle
newRoute;320;bjictfniafzvvefu;dawcsmfxldvkjmntwrrgrdknaqle
addRoad;bjictfniafzvvefu;dawcsmfxldvkjmntwrrgrdknaqle;17;-8
extendRoute;294;dawcsmfxldvkjmntwrrgrdknaqle
addRoad;bjictfniafzvvefu;bjictfniafzvvefu;12;5
extendRoute;293;rdebljwrwetdgqakyvnuv
addRoad;kegxiwchvwecbjvjpvppuqfixwof;hccmkzuidd;28;-14
repairRoad;dawcsmfxldvkjmntwrrgrdknaqle;xsxhfdsggagzxyoofregsr;10
getRouteDescription;176
newRoute;211;bjictfniafzvvefu;yloqnkncpf
repairRoad;dawcsmfxldvkjmntwrrgrdknaqle;xsxhfdsggagzxyoofregsr;12
repairRoad;yloqnkncpf;zzhrpgfmjyrqbdzxwxerb;9
repairRoad;dawcsmfxldvkjmntwrrgrdknaqle;kegxiwchvwecbjvjpvppuqfixwof;11
extendRoute;873;dawcsmfxldvkjmntwrrgrdknaqle
extendRoute;211;zzhrpgfmjyrqbdzxwxerb
addRoad;yloqnkncpf;dawcsmfxldvkjmntwrrgrdknaqle;9;-3
addRoad;watdcwroxox;yloqnkncpf;24;2
repairRoad;rdebljwrwetdgqakyvnuv;kdwiymevvdwewm;-13
newRoute;662;zzhrpgfmjyrqbdzxwxerb;zzhrpgfmjyrqbdzxwxerb
removeRoad;kegxiwchvwecbjvjpvppuqfixwof;hccmkzuidd
removeRoad;hccmkzuidd;kegxiwchvwecbjvjpvppuqfixwof
extendRoute;662;bjictfniafzvvefu
repairRoad;rdebljwrwetdgqakyvnuv;rdebljwrwetdgqakyvnuv;7
removeRoad;yloqnkncpf;dawcsmfxldvkjmntwrrgrdknaqle
removeRoad;dawcsmfxldvkjmntwrrgrdknaqle;bjictfniafzvvefu
addRoad;kegxiwchvwecbjvjpvppuqfixwof;rdebljwrwetdgqakyvnuv;19;15
repairRoad;rdebljwrwetdgqakyvnuv;xsxhfdsggagzxyoofregsr;-1
extendRoute;662;kdwiymevvdwewm
repairRoad;xsxhfdsggagzxyoofregsr;xsxhfdsggagzxyoofregsr;6
getRouteDescription;294
addRoad;yloqnkncpf;xsxhfdsggagzxyoofregsr;14;2
repairRoad;kegxiwchvwecbjvjpvppuqfixwof;kegxiwchvwecbjvjpvppuqfixwof;6
removeRoad;xsxhfdsggagzxyoofregsr;kdwiymevvdwewm
removeRoad;kegxiwchvwecbjvjpvppuqfixwof;hccmkzuidd
