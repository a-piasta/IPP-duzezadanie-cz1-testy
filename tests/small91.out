1: NIE
2: TAK
3: NIE
4: NIE
5: TAK
6: 
7: NIE
8: TAK
9: NIE
10: NIE
11: 
12: NIE
13: NIE
14: 
15: TAK
16: NIE
17: NIE
18: 
19: NIE
20: NIE
21: NIE
22: 
23: NIE
24: NIE
25: NIE
26: TAK
27: NIE
28: NIE
29: TAK
30: NIE
31: NIE
32: NIE
33: NIE
34: NIE
35: 
36: NIE
37: NIE
38: NIE
39: NIE
40: NIE
41: NIE
42: TAK
43: TAK
44: NIE
45: NIE
46: TAK
47: NIE
48: NIE
49: NIE
50: TAK
51: TAK
52: TAK
53: NIE
54: NIE
55: NIE
56: 
57: TAK
58: NIE
59: TAK
60: NIE
